xml/System.Numerics/Vector`1.xml

```diff
@@ -57,7 +57,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">The vector type. <typeparamref name="T" /> can be any primitive numeric type.</typeparam>
+    <typeparam name="T">The type of the elements in the vector. <typeparamref name="T" /> can be any primitive numeric type.</typeparam>
```

Comment:
@tannergooding FYI the `remarks` for this specific case defines what they mean by "primitive numeric type". I removed the term from the other files that don't have that context.

Reply:
Would probably be good to update the remarks to cover `nint` and `nuint`.

The general table there is also out of date since its really only covering the semantics around `x86/x64` and not other platforms such as `Arm64`.

Reply:
Would removing the table be the best course of action in that case (see my new commit)? The rest of the remarks section is general enough to apply to both architectures. I suppose it depends on how easy it would be for users to figure out those types with other sources.

Reply:
You can remove the remarks table in this PR and resubmit it without urgency on a later date.

Remarks do not show up in intellisense, so they aren't urgent for getting them merged today.

Reply:
Sounds good. I'll merge this now.